modules/@apostrophecms/translation/index.js

```diff
@@ -6,21 +6,23 @@ module.exports = {
   },
   init(self) {
     self.providers = [];
-
     self.enableBrowserData();
   },
   handlers(self) {
     return {
       '@apostrophecms/doc-type:beforeLocalize': {
         // Translate the document using the first available provider.
         async translate(req, doc, {
-          source, target, docType, existing
+          source, target, existing
         }) {
+          if (!self.isEnabled()) {
+            return;
+          }
           const targets = req.query.aposTranslateTargets || [];
-          const providerName = self.apos.launder.string(req.query.aposTranslateProvider);
+          const aposProvider = self.apos.launder.string(req.query.aposTranslateProvider);
```

Comment:
Should be used directly when retrieving provider and provider module to avoid bugs when chaining the results of those.